.github/workflows/ci.yml

```diff
@@ -290,3 +290,36 @@ jobs:
         with:
           name: notebook-outputs
           path: out
+  ts-lint:
+    name: Typescript lint check
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          node-version: '14.16.1'
+      - name: Install npm
+        run: npm --prefix 'cirq-web/cirq_ts' ci
+      - name: Lint
+        run: check/ts-lint 
+  ts-test:
+    name: Typescript unit testing
```

Comment:
This is not just unit test, but also the e2e tests, let's call this `Typescript tests`.